app/modules/ads/slots.js

```diff
@@ -379,6 +379,26 @@ export const slots = {
       context.set('slots.incontent_boxad_1.targeting.xna', '0');
     }
   },
+
+  setupCloseButtonTimer() {
+    const { context } = window.Wikia.adEngine;
+
+    context.push('listeners.slot', {
+      onRenderEnded: (adSlot) => {
+        if (['invisible_high_impact_2', 'floor_adhesion'].includes(adSlot.getSlotName())) {
+          const closeButton = document.querySelector('.out-of-page-template.floor-adhesion .button-close');
+
+          if (closeButton) {
+            closeButton.style.display = 'none';
```

Comment:
Changing styles in JS may not be a good practice. But having a class in CSS which we switch is probably more bytes ;) And it's an experiment, anyway.

Reply:
Absolute right with this.